docs/releasenotes.md

```diff
@@ -0,0 +1,83 @@
+# Releasenotes Haal-Centraal-BRP-historie-bevragen
+
+
+## Versie 1.0.0
+
+Alhoewel er nog geen eerdere officiële versie van de Haal-Centraal-BRP-historie-bevragen API als release is uitgebracht zijn de specificaties van deze endpoints als onderdeel van de specificatie van Haal-Centraal-BRP-bevragen het afgelopen jaar beschikbaar geweest via de repository. In de [v0.9.0 release van BRP-bevragen](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/tree/v0.9.0) zijn de historie-endpoints nog opgenomen. In de afgelopen maanden hebben we de historie-endpoints in een aparte API ondergebracht. Daarbij zijn nieuwe inzichten die het afgelopen jaar zijn opgedaan opgenomen. Recentelijk hebben we de specificaties ook aangepast op het hergebruik van componenten uit BRP-Bevragen release 1.2.0.
+Dit is een globaal (en niet limitatief) overzicht van wijzigingen die daarvoor zijn doorgevoerd.
+
+### Openapi.yaml :
+
+- server-url is aangepast (omdat het een aparte API is geworden.) --> https://github.com/VNG-Realisatie/Haal-Centraal-BRP-historie-bevragen _**(breaking)**_
+- OperationId's zijn aangepast ivm code-generatie issues en nu in UpperCamelCase. _**(breaking)**_
+- Alle descriptions zijn omgezet naar markdown.
```

Comment:
waarom niet de belangrijkste wijzigingen (alles wat in de specificaties is gewijzigd, te beginnen met breaking changes) bovenaan?
Ik betwijfel zelfs of wijzigingen in de documentatie in release notes horen (maar nu niet verkeerd zodat een gebruiker die een diff doet snapt waarom er zoveel wijzigingen zijn)

Reply:
Dat is een keuze in welke groepering je aanhoudt. Ik heb alles wat breaking is aangemerkt met een bold en cursief lettertype.   Als je dit in een markdown viewer bekijkt springt het er behoorlijk uit.  Heeft met smaak te maken denk ik.

Reply:
En de documentatie (bedoel je daar de features mee ?) Kan heel relevant zijn voor de partij die de provider implementeert. Dat hoeft niet breaking te zijn voor consumers, maar wel zeer belangrijk voor de providers.

Reply:
met documentatie bedoel ik alles dat geen directe consequenties heeft voor de werking van de API, zoals descriptions

Reply:
Dat is 1 zin waarmee ik een heleboel verschillen verklaar als een developer een diff doet van de 2 versies van de openapi.yaml.  Dat vindt ik wel toegevoegde waarde hebben, maar ik ben niet de gebruiker van deze releasenotes. @Melvin, wat denk jij ?

Reply:
ik ben dat wel met je eens, alleen zou ik (ik herhaal mezelf) dingen waar developers echt iets voor moeten doen bovenaan zetten, en tekstuele/visuele wijzigingen onderaan. Maar als jij een goed idee hebt waarom deze groepering is gekozen, ook goed.

Reply:
Ik was en ben nu nog steeds van mening dat het wijzigen van documentatie (ook in de OAS yaml bestanden) ten behoeve van leesbaarheid en beter begrip van functionaliteit geen wijziging is van functionaliteit.

Het alleen verbeteren van documentatie mag dus naar mijn mening niet leiden tot een nieuwe versie en dus ook niet tot een nieuwe release. Een release note maak je alleen aan voor een nieuwe release, dus wat mij betreft hoeft verbetering van documentatie als punt niet te worden opgenomen in een release note.

Reply:
documentatiewijzigingen worden niet in de readme opgenomen.